### ISSUE TITLE
FC from user reports - MainView.java:471

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#166]
```
Taken from submitted user reports through Android Market. This is a FC that has 
happened on some users' devices.


v0.6.7
Feb 24, 2011 11:07:32 PM
2 reports/week
0%
11 reports
java.lang.RuntimeException: Unable to resume activity 
{com.gpl.rpg.AndorsTrail/com.gpl.rpg.AndorsTrail.activity.MainActivity}: 
java.lang.NullPointerException
at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3128)
at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3143)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2684)
at android.app.ActivityThread.access$2300(ActivityThread.java:125)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2033)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:123)
at android.app.ActivityThread.main(ActivityThread.java:4627)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:521)
at 
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:858)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.NullPointerException
at com.gpl.rpg.AndorsTrail.view.MainView.notifyMapChanged(MainView.java:471)
at 
com.gpl.rpg.AndorsTrail.controller.Controller.handlePlayerDeath(Controller.java:
130)
at 
com.gpl.rpg.AndorsTrail.controller.CombatController.handleNextMonsterAction(Comb
atController.java:301)
at 
com.gpl.rpg.AndorsTrail.controller.CombatController.endPlayerTurn(CombatControll
er.java:250)
at 
com.gpl.rpg.AndorsTrail.controller.CombatController.maybeAutoEndTurn(CombatContr
oller.java:210)
at 
com.gpl.rpg.AndorsTrail.controller.CombatController.enterCombat(CombatController
.java:53)
at com.gpl.rpg.AndorsTrail.activity.MainActivity.onResume(MainActivity.java:237)
at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1149)
at android.app.Activity.performResume(Activity.java:3823)
at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3118)
... 12 more



Comments left by users:

Feb 24, 2011 11:07:32 PM
0.6.7
does not work
Feb 17, 2011 7:52:11 PM
0.6.7
can't open
Feb 6, 2011 10:20:31 PM
0.6.7
I turned the screen and it forced close, and won't restart
Jan 9, 2011 12:07:29 AM
0.6.7
after hitting continue current game it forces closed
Dec 31, 2010 3:24:45 PM
0.6.7
the game is great. this is the first force close. it occured while I wad 
cleaning the screen mid-game.
Dec 25, 2010 10:03:56 AM
0.6.7
add keeps freezing
Dec 25, 2010 7:39:57 AM
0.6.7
a force close was required when I tried to fight the big rat.
Dec 24, 2010 4:04:55 AM
0.6.7
wont load my game
```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 24 Feb 2011 at 10:14

**Comment by GoogleCodeExporter** [comment]
```
Note that this was submitted from players having v0.6.7 , so this bug must have 
been present for quite a while.
```

Original comment by `oskar.wi...@gmail.com` on 24 Feb 2011 at 10:16

**Comment by GoogleCodeExporter** [comment]
```
Should be resolved with the changes done for issue 274.
```

Original comment by `oskar.wi...@gmail.com` on 29 Apr 2012 at 3:58
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
